testing/checkin/runtime/async.ts

```diff
@@ -89,6 +89,16 @@ export class StatsCollection {
     return count;
   }
 
+  countResources(): number {
+    let count = 0;
+    for (const item of this.data) {
+      if ("Resource" in item) {
```

Comment:
Where is thing magic string coming from? Could you add a comment describing it?

Reply:
These come from `ResourceActivity` which is `#[serde]` return type from these ops. Note that these are checkin ops so this won't be in Deno itself. That code will be pure Rust.

I'll add some notes pointing at what these types are.